hallux/backends/dummy_backend.py

```diff
@@ -37,8 +37,8 @@ def __init__(
             self.save_on_exit = True
 
     def issue_hash(self, description: str, issue_lines: list[str]) -> str:
-        bytes = str(description + "\n" + "\n".join(issue_lines)).encode("utf8")
-        return str(hashlib.md5(bytes).hexdigest())
+        encoded_data = str(description + "\n" + "\n".join(issue_lines)).encode("utf8") 
```

Comment:
why `encoded_data` ?

Reply:
open for a better variable name, there was a sonar issue complaining regarding bytes: 

> Rename this variable; it shadows a builtin.